docker/Dockerfile

```diff
@@ -312,6 +310,9 @@ USER nautobot
 WORKDIR /opt/nautobot
 
 COPY --from=final-dev --chown=nautobot /opt/nautobot/nautobot_config.py /opt/nautobot/nautobot_config.py
+RUN mkdocs build --strict=False -q
+# Copy in the built docs to bundle with the wheel
+COPY /source/nautobot/project-static/docs /source/nautobot/project-static/docs
```

Comment:
This doesn't seem right to me?
1) Same source and target paths?
2) Wheel has already been built at this point so this is too late to bundle in, in any case.